packages/next/src/server/config-shared.ts

```diff
@@ -699,6 +699,11 @@ export interface ExperimentalConfig {
    * Enable segment viewer for the app directory in dev tool.
    */
   devtoolSegmentExplorer?: boolean
+
+  /**
+   * Enable new panel UI for the dev tool.
```

Comment:
```suggestion
   * Enable new panel UI for Next DevTools.
```

There's no singular dev tool. This branding also matches other similar tools (React DevTools, Chrome DevTools).

The flag naming should be kept to be in line with the segment explorer. If we ever introduce a new devtools flag for the first time, it should be plural.